packages/mosaic/timepicker/timepicker.spec.ts

```diff
@@ -594,5 +596,37 @@ describe('McTimepicker', () => {
                     expect(inputElementDebug.nativeElement.value).toBe('23:59', 'Arrow-Down key decrement not working');
                 });
         });
+
+        it('Manual keyboard input digit-by-digit', () => {
+            return fixture.whenStable()
+                .then(() => {
+                    dispatchFakeEvent(inputElementDebug.nativeElement, 'focus');
+                    inputElementDebug.nativeElement.selectionStart = 0;
+                    inputElementDebug.nativeElement.selectionEnd = 2;
+
+                    const key1PressEvent: KeyboardEvent = createKeyboardEvent('keydown', ONE);
+                    dispatchEvent(inputElementDebug.nativeElement, key1PressEvent);
+                    inputElementDebug.nativeElement.value = `1${inputElementDebug.nativeElement.value.substring(2)}`;
+                    inputElementDebug.nativeElement.selectionStart = 1;
+                    inputElementDebug.nativeElement.selectionEnd = 1;
+                    dispatchFakeEvent(inputElementDebug.nativeElement, 'input');
+                    fixture.detectChanges();
+
+                    expect(inputElementDebug.nativeElement.value).toBe('1:00', 'Keypress works!');
+
+                    const key2PressEvent: KeyboardEvent = createKeyboardEvent('keydown', TWO);
+                    dispatchEvent(inputElementDebug.nativeElement, key2PressEvent);
+                    inputElementDebug.nativeElement.value =
+                        `${inputElementDebug.nativeElement.value.substring(0, 1)}` +
```

Comment:
why do you use concatenating template literals ? in this case you should use only template literals.

Reply:
I need to produce single-line string, but template literal with line breaks produces string with line breaks.

Template literals without line-breaks causes long line (>120 symbols) in source code.

Using simple strings with concatenation causes tslint to warn `prefer template strings`.

Reply:
I've splitted this string on two variables and then used single template string.